### ISSUE TITLE
Plain text not working for {plain text} in Sublime Text 3114

### ISSUE THREAD

**Issue by methodbox** [sergeche/emmet-sublime#625]
Pulling directly from the Emmet documentation:

```
p>{Click }+a{here}+{ to continue}
```

...produces

```
<p>Click <a href="">here</a> to continue</p>
```

...except it doesn't. It produces this:

```
<p>Click <a href="">here</a></p>
```

I discovered this trying to create a quick `script` tag for use with Vuejs:

```
"script:v": "<script>{some text and brackets and stuff}</script>
```

but it came out like this:

```
<script></script>
```

Note: just for the hell of it I tried both Tab and Ctrl+E but the "expansion" appears the same - not working correctly.

**Comment by zen77** [comment]
I have the same problem. I can't find a fix either.

**Comment by methodbox** [comment]
I went back to Sublime 2 and enabled high resolution fonts (which was really the thing that prompted my move to 3) and can edit the completions files directly by just browsing packages.

I don't find 3 nearly as easily hackable as 2 and 3 eats more battery and CPU on my Mac by about 10%, not to mention it's no faster. I see no advantage.

I've been able to create plenty of triggerable HTML and JavaScript blocks in minutes. The syntax is simpler than Emmet and doesn't require a restart of Sublime to take effect.

Just like with Atom, until I actually see an advantage I am sticking with old faithful.

The best argument I've seen for Atom is running other connections like Git and FTP or an app server like node, all of which can easily be done in Sublime with a quick Google search.
